index.bs

```diff
@@ -4558,6 +4558,23 @@ FIDO APIs use an alternative identifier for [=[RPS]=] called an |AppID|
 that identifier. Without this extension, they would need to be re-registered in
 order to be [=scoped=] to an [=RP ID=].
 
+In addition to setting the {{AuthenticationExtensionsClientInputs/appid}} extension input,
+using this extension requires some additional processing by the [=[RP]=]
+in order to [=authentication|authenticate=] U2F credentials:
+
+ 1. List the desired U2F credentials in the {{PublicKeyCredentialRequestOptions/allowCredentials}} option
+     of the {{CredentialsContainer/get()}} method:
+
+     - Set the {{PublicKeyCredentialDescriptor/type}} members to {{PublicKeyCredentialType/public-key}}.
+     - Set the {{PublicKeyCredentialDescriptor/id}} members to the respective U2F key handles of the desired credentials.
+
+     {{PublicKeyCredentialRequestOptions/allowCredentials}} MAY contain a mixture
+     of both WebAuthn [=credential IDs=] and U2F key handles;
+     setting the {{AuthenticationExtensionsClientInputs/appid}} does not prevent the user from using a WebAuthn credential.
```

Comment:
suggest:
stating the {{AuthenticationExtensionsClientInputs/appid}} via this extension does not prevent the user from using a WebAuthn-registered credential that is scoped to the [=RP ID=] stated in  {{PublicKeyCredentialRequestOptions/rpId}}.